.release.cloudbuild.yaml

```diff
@@ -398,8 +406,8 @@ steps:
   - |
     docker tag gcr.io/$PROJECT_ID/metadata-envoy:$COMMIT_SHA gcr.io/ml-pipeline/google/pipelines/metadataenvoy:$(cat /workspace/mm.ver)
     docker tag gcr.io/$PROJECT_ID/metadata-envoy:$COMMIT_SHA gcr.io/ml-pipeline/google/pipelines-test/metadataenvoy:$(cat /workspace/mm.ver)
-    docker push gcr.io/ml-pipeline/google/pipelines/metadataenvoy:$(cat /workspace/mm.ver)
-    docker push gcr.io/ml-pipeline/google/pipelines-test/metadataenvoy:$(cat /workspace/mm.ver)
+    docker push gcr.io/ml-pipeline/google/pipelines/metadata-envoy:$(cat /workspace/mm.ver)
```

Comment:
Sorry, I might have not been clear enough.
For standalone deployment images, we should use `-` separated.
For marketplace images, there cannot be `-`s in them.

These are below /google/ directory, so they are marketplace images

Reply:
Sorry I think it my mistake, I didn't read the path correctly. I have fixed it now.